pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1995,3 +1995,56 @@ func makeQueuedPodInfos(num int, timestamp time.Time) []*framework.QueuedPodInfo
 	}
 	return pInfos
 }
+
+func TestPriorityQueue_calculateBackoffDuration(t *testing.T) {
+	type args struct {
+		podInfo *framework.QueuedPodInfo
+	}
+	tests := []struct {
+		name  string
+		queue *PriorityQueue
```

Comment:
Replace it with `initialBackoffDuration` and `maxBackoffDuration`, and then instantiate the queue object in the testing block.